src/components/Validation/Validation.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import classNames from 'classnames';
+import { bindClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition } from '../../util/component-definition';
+import _ from 'lodash';
+
+const boundClassNames = bindClassNames('Validation');
+
+const {
+	any,
+} = React.PropTypes;
+
+/**
+ *
+ * {"categories": ["helpers"]}
+ *
+ * Validation is a wrapper component that's meant to be used by other
+ * components. Wrap your form components in it and style them accordingly if
+ * there's an error.
+ */
+const Validation = React.createClass(createLucidComponentDefinition({
+	displayName: 'Validation',
+
+	childProps: {
+		Error: {} // intentionally left blank since we only care about `children`
```

Comment:
we can probably make this use case better in the component definition, `null` also works here, but might look weird